### PR TITLE
docs: recipe to debug with DevTools

### PR DESCRIPTION
**Pull request by mightyiam**
Please do not mark this as a fix for #1112.
I have another PR coming for that.

### CODE REVIEW

File: docs/recipes/debugging-with-chrome-devtools.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Debugging Tests with Chrome WebTools
+
+Use [inspect-process](https://www.npmjs.com/package/inspect-process)
+to easily launch a debugging session with Chrome DevTools.
```
Comment (sindresorhus) [inline docs/recipes/debugging-with-chrome-devtools.md]: Don't hard-wrap

File: docs/recipes/debugging-with-chrome-devtools.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+``` console
```
Comment (sindresorhus) [inline docs/recipes/debugging-with-chrome-devtools.md]: Should be no space before `console`.

**Comment by mightyiam** [comment]
The changes requested by @sindresorhus were committed.

**Comment by sindresorhus** [comment]
Thank you :)

**Pull request merged**